src/shared/paths.enum.js

```diff
@@ -0,0 +1,10 @@
+export const paths = Object.freeze({
+    USER_PANEL: '/user_panel',
+    MY_DEPARTMENT: '/my_department',
+    DEPARTMENTS_BASE: '/departments_base',
+    MANAGERS_BASE: '/managers_base',
+    LIBRARIAN_BASE: '/librarian_base',
+    READERS_BASE: '/readers_base',
+    BOOKS_BASE: '/books_base',
+    LOGIN: '/login',
```

Comment:
Continuing the idea with decentralized routers, here in paths could be also implemented nesting by `authPaths` and `dashboardPaths` and even deeper if needed...

... and it could also be moved to respective subdirectory